docs/guides/dynamic-circuits-considerations.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Hardware considerations and limitations
+description: Hardware considerations and limitations.  Circuits with mid-circuit measurements and dynamic circuits are most likely to hit these limits.
+
+---
+# Hardware considerations and limitations
+
+Running circuits on quantum processors involves not only the qubits themselves, but also a system of classical electronics and computers to generate and receive waveforms and orchestrate the control logic. There are physical limitations on these classical hardware coomponents that, when exceeded, would cause the job to fail. Jobs that contain a large number of classical control flows or measurements are most susceptible to these limitations due to the amount of data that needs to be fetched and transferred.
```

Comment:
```suggestion
Running circuits on quantum processors involves not only the qubits themselves, but also a system of classical electronics and computers to generate and receive waveforms and orchestrate the control logic. There are physical limitations on these classical hardware components that, when exceeded, would cause the job to fail. Jobs that contain a large number of classical control flows or measurements are most susceptible to these limitations due to the amount of data that needs to be fetched and transferred.
```